src/modules/common/utils/featuredImageToSrcSet.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { FeaturedImageModel } from '@integreat-app/integreat-api-client'
+
+const ten = 10
+const roundToOneDecimal = number => Math.round(number * ten) / ten
+
+/**
+ * This converts a FeaturedImageModel to a scrset.
```

Comment:
Maybe add a link to information about srcsets.

Reply:
Hmm... as this is a standard-attribute, I think we should not add a link...  I'll fix the typo though :sweat_smile:

Reply:
Hmm... as this is a standard-attribute, I think we should not add a link...  I'll fix the typo though :D